### DISCUSSION TITLE
Using Sentinel ETAD datasets

### DISCUSSION THREAD

**Discussion by Pinoquio1891** [isce-framework/isce2#846] [Q&A]
Hi!
Currently we are working with corner reflectors and using ISCE in combination with MiaPlPy to analyze the time series signal of the reflectors. In order to further improve the results using Sentinel-1 signal corrections we noticed the 'new service' from ESA, the ETAD datasets (https://sentinels.copernicus.eu/web/sentinel/missions/sentinel-1/data-products/etad-dataset). In order to use these datasets the phase corrections obtained from ETAD should be aligned with the different phasemaps of the interferogram. Looking at the ISCE code we noticed that the co-registration process is pretty elaborated and we were wondering, if there is an easy way to extract the co-registration information from ISCE to use them with the ETAD data.
Cheers
Christian

**Comment by EJFielding** [comment]
Hi Christian,

The developers who know the details about the ISCE2 processing are busy with development of ISCE3 for NISAR and OPERA processing, so I don't think they are reading questions here. I am not a developer, so I can't answer the details of your question.

You might want to look at the ISCE3 code that was completely rewritten from ISCE2 and may be easier to read.

++Eric Fielding

**Reply by Pinoquio1891** [reply]
Hi Eric,

thanks for your reply. For now I decided to stay with isce2, as, after a short look at isce3 repository was in doubt if the Sentinel-1 part is already implemented there. I guess for now I solved most of my problems, though there are still some issues that might have to be solved. Currently I did not separate bursts into upper and lower, as isce does, I will have to check about the consequences of this.

Cheers Christian

**Reply by EJFielding** [reply]
Hi Christian,

Yes, the ISCE3 software fully supports processing Sentinel-1 data. The OPERA project at JPL is using it for their systematic processing of Sentinel-1 data. They have created a simplified workflow to use the ISCE3 library for processing Sentinel-1 called "sweets": https://github.com/isce-framework/sweets
++Eric

**Reply by vincentschut** [reply]
Interesting. Slightly related to this (sorry for hijacking): I wondered if ISCE3 also supports Sentinel-1 GRD instead of SLC data? We currently use ISCE2 to preprocess S1 GRD data; if ISCE3 already supports that, we might consider testing/upgrading our workflow to ISCE3... @EJFielding do you know is ISCE3 also supports S1 GRD data?

**Reply by EJFielding** [reply]
@vincentschut I don't know if ISCE3 supports processing S1 GRD files. I would ask on the ISCE3 repository https://github.com/isce-framework/isce3
